presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -324,6 +324,16 @@ public void testSchemaEvolution()
         testWithAllFileFormats(this::testSchemaEvolution);
     }
 
+    @Test
+    public void testInsertIntoNotNullColumn()
+    {
+        assertUpdate("CREATE TABLE test_not_null_table (c1 INTEGER, c2 INTEGER NOT NULL)");
+        assertUpdate("INSERT INTO test_not_null_table (c2) VALUES(2)", 1);
```

Comment:
For SQL formatting, add spaces before opening parenthesis: `VALUES (2)`

Reply:
Done in the latest force push.